### ISSUE TITLE
Finetune CogVideoX 1.5 (5B) I2V on A100 40GB

### ISSUE THREAD

**Issue by royvelich** [a-r-r-o-w/finetrainers#152]
Hi,
Should it be possible to finetune CogVideoX 1.5 (5B) I2V on a single or multiple A100 40GB? Or is it mandatory to have A100 80GB?

**Comment by a-r-r-o-w** [comment]
Hey! It should be possible to finetune it in under 24 GB for a batch size of 1, gradient checkpointing and using precomputed latents and prompt embeddings. This has been verified by a few folks already, so please do let me know if there's any problem in getting it to work under 40 GB
